mandatory/debugging-practice/script.js

```diff
@@ -38,7 +38,7 @@ function submit() {
     return false;
   } else {
     let book = new Book(title.value, title.value, pages.value, check.checked);
```

Comment:
Overall looks like you have got most of the bugs, but thorough testing is important! What is wrong with this line?